Exygy/askdarcel-web
# Listing page layout

# Pull Request Template

## Issue Overview

This PR addresses [Organization Detail Page](https://www.notion.so/exygy/Organization-Detail-Page-001a835af95944c0b61b72318ead2bff) and [Organization Page Styling](https://www.notion.so/exygy/Organization-Page-Styling-1592ec01023745058c480be1ed11c905)


## Description

This PR is refactors the Organization and Styling pages to use the same reusable ListingPageWrapper component, and refactors global css into component module scss files, cleaning up the codebase.

## How Can This Be Tested/Reviewed?

Visit any /organization/ORGID and /service/SRVID page and compare to mockup and each other. Should share styling now. Check code for improvements.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
